src/main/java/org/ibp/api/brapi/v1/study/StudyResourceBrapi.java

```diff
@@ -105,39 +107,73 @@
 	@Autowired
 	InstanceValidator instanceValidator;
 
-	@ApiOperation(value = "List of study summaries", notes = "Get a list of study summaries.")
-	// TODO implement
-	// @RequestMapping(value = "/{crop}/brapi/v1/studies", method = RequestMethod.GET)
+	@ApiOperation(value = "List of studies", notes = "Get a list of studies.")
+	@RequestMapping(value = "/{crop}/brapi/v1/studies", method = RequestMethod.GET)
 	@ResponseBody
-	public ResponseEntity<StudySummariesDto> listStudySummaries(@PathVariable final String crop,
+	public ResponseEntity<EntityListResponse<StudyDto>> listStudies(@PathVariable final String crop,
+		@ApiParam(value = "Common name for the crop associated with this study.") @RequestParam(value = "commonCropName", required = false)
```

Comment:
Just a comment, maybe on the brapi spec.. It doesn't seem useful to include "commonCropName" as API parameter given we've already set the crop context in URL
cc: @nahuel-soldevilla

Reply:
We were following the [convention of accepting the parameter but then ignoring it](https://ibplatform.atlassian.net/browse/IBP-1841?focusedCommentId=35035&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-35035).

@darla-leafnode @abatac